drivers/mongodb-sync-v4-driver/src/main/java/com/github/cloudyrock/mongock/driver/mongodb/sync/v4/changelogs/runalways/MongockSync4LegacyMigrationChangeRunAlwaysLog.java

```diff
@@ -0,0 +1,25 @@
+package com.github.cloudyrock.mongock.driver.mongodb.sync.v4.changelogs.runalways;
+
+import com.github.cloudyrock.mongock.ChangeLog;
+import com.github.cloudyrock.mongock.ChangeSet;
+import com.github.cloudyrock.mongock.driver.mongodb.sync.v4.changelogs.LegacyService;
+import com.github.cloudyrock.mongock.migration.MongockLegacyMigration;
+import com.mongodb.client.MongoDatabase;
+import io.changock.driver.api.entry.ChangeEntry;
+import io.changock.driver.api.entry.ChangeEntryService;
+import io.changock.migration.api.annotations.NonLockGuarded;
+import io.changock.migration.api.annotations.NonLockGuardedType;
+
+import javax.inject.Named;
+
+@ChangeLog(order = "00001")
+public class MongockSync4LegacyMigrationChangeRunAlwaysLog {
```

Comment:
Minor, but it seems a little awkward to split `ChangeLog` in the name, since a "change log" is a single concept. Perhaps `MongockSync4LegacyMigrationRunAlwaysChangeLog` would be a more natural name?

Reply:
hahaha, of course that wasn't the idea. It should have been like you say. It will be changed, but not urgent, it's internal, doesn't affect to users ;)